root/artist/edit_form.tt

```diff
@@ -73,7 +73,13 @@
       <fieldset>
         <legend>[% l('Artist Credits') %]</legend>
         <p>
-            [%- l('Please select the artist credits that you want to rename to follow the new artist name.') -%]
+            [%- l('Please select the {doc|artist credits} that you want to rename to follow the new artist name.', {doc => doc_link('Artist Credits')}) -%]
```

Comment:
Didn't want to break the existing translation, but it seems like this really should link to what ACs actually *are*.